sentry_sdk/integrations/stdlib.py

```diff
@@ -61,6 +64,14 @@ def getresponse(self, *args, **kwargs):
         if "status_code" not in data:
             data["status_code"] = rv.status
             data["reason"] = rv.reason
+
+        span = self._sentrysdk_span
+        if span is not None:
+            span.set_tag("status_code", rv.status)
```

Comment:
This could be `http.status_code`. Again OT

Reply:
fixed